discretize/utils/meshutils.py

```diff
@@ -255,3 +258,426 @@ def ExtractCoreMesh(xyzlim, mesh, meshType='tensor'):
         raise Exception("Not implemented!")
 
     return actind, meshCore
+
+
+def meshBuilderXYZ(
+    xyz, h,
+    padX=[0, 0], padY=[0, 0], padZ=[0, 0],
+    baseMesh=None,
+    expFact=1.3,
+    meshType='TENSOR',
+    verticalAlignment='top'
+):
```

Comment:
The naming convention states `camelCase` for classes, but `lower_case_with_underscores` for functions and variables (https://www.python.org/dev/peps/pep-0008/#function-and-variable-names). Even though SimPEG is not strict in this, I think it would be preferred that new code follows it. Hence `def mesh_builder_xyz`, and `pad_x`, `exp_fact`, `mesh_type`, `vertical_alignment` etc.

Reply:
@fourndo: would you mind taking a look at @prisae's comments wrt naming here?